force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -176,7 +199,10 @@ public with sharing class VariablesDataTypesOperators {
      * @return The average of the three numbers.
      */
     public static Double averageOfThreeNumbers(Integer a, Integer b, Integer c) {
-        return null; // Replace null with the variable you used to store the result
+        Integer sum = a + b + c;
+        Integer totalNumbers = 3;
+        Double average = sum / totalNumbers;
```

Comment:
what happens when the sum is not perfectly divisible by 3? Integer / Integer always return a whole number, even if declare the variable a Double because of the inputs.